src/storage/hummock_test/src/bin/replay/main.rs

```diff
@@ -132,6 +132,7 @@ async fn create_replay_hummock(r: Record, args: &Args) -> Result<impl GlobalRepl
         max_prefetch_block_number: storage_opts.max_prefetch_block_number,
         recent_filter: None,
         state_store_metrics: state_store_metrics.clone(),
+        use_new_object_prefix_strategy: false,
```

Comment:
let's make this part of the Args: https://github.com/risingwavelabs/risingwave/blob/380912047757c25628e82c69e47a7fe8ef7a4c6d/src/storage/hummock_test/src/bin/replay/main.rs#L55